src/Stores/SubjectsPromises/reducer.js

```diff
@@ -20,9 +21,16 @@ const subjectsPromisesReducer = (state: State = initialState, action: Action): S
     case ActionTypes.SUBJECTS_PROMISES_LOAD:
       return { ...state, loading: true, data: null, error: null };
     case ActionTypes.SUBJECTS_PROMISES_LOAD_SUCCESS:
-      return { ...state, loading: false, data: action.subjects };
+      return { ...state, loading: false, data: action.data };
     case ActionTypes.SUBJECTS_PROMISES_LOAD_FAILURE:
       return { ...state, loading: false, error: action.error };
+    case ActionTypes.SUBJECT_PROMISE_UNLINK_SUCCESS: {
+      const filteredSPs = filter(state.data, sp => sp.id !== action.id);
```

Comment:
not better to just user state.data.filter(...)?

Reply:
yes it is. Fixed :)